ci-operator/jobs/openshift/origin/openshift-origin-master-presubmits.yaml

```diff
@@ -17,6 +17,8 @@ presubmits:
         - --artifact-dir=$(ARTIFACTS)
         - --branch=master
         - --give-pr-author-access-to-namespace=true
+        - --image-import-pull-secret=/etc/pull-secret/.dockerconfigjson
```

Comment:
We keep this job on api.ci because 6G of Mem is going to hurt our scale up as discussed.
And the rehearsal is indeed on default:
https://prow.svc.ci.openshift.org/prowjob?prowjob=905d78bf-4789-11ea-b42d-0a58ac103b5d